aws/table_aws_organizations_account.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsOrganizationsAccount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_organizations_account",
+		Description: "AWS Organizations Account",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"AccountNotFoundException"}),
+			Hydrate:           getOrganizationsAccount,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listOrganizationsAccounts,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The unique identifier (ID) of the account.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the account.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "email",
+				Description: "The email address associated with the AWS account.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "joined_method",
+				Description: "The method by which the account joined the organization.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "joined_timestamp",
+				Description: "The date the account became a part of the organization.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "name",
+				Description: "The description of the permission set.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The status of the account in the organization.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "tags",
```

Comment:
The tags on this resource, as a map of key:value pairs. Many resources support tags, though not all in the same format. If the provider tags are in a different format, expose them in the native format in a tags_src column, and convert them to key:value map in the tags column. When tags are simple labels with no key:value (like github issue lables), use the format label:true.  For reference - aws_cloudwatch_alarm table.

Reply:
Done. I note there's a few tables that don't follow this standard though (e.g. `aws_accessanalyzer_analyzer`). I'm also interested to know the reason for doing this? As a user it doesn't seem to add any value - the `tags_src` field is less concise/readable whilst duplicating the standard `tags` field.

Reply:
In `aws_accessanalyzer_analyzer` table, we get tags property as `{key:value}` format in API response, which is steampipe standard format we use, so there is no need to have a separate column. Whereas, in case of organization the tags property value comes as `[{Key: test, Value: testvalue}]` format and we keep the raw response in tags_src column and provide `{test : testvalue}` in tags as steampipe standard column.

On the last question, whether it adds value or not, we consider this as a suggestion. We will discuss it internally.